### ISSUE TITLE
Add Fedora URI as link header

### ISSUE THREAD

**Issue by dannylamb** [Islandora/documentation#880]
Do a lookup in Gemini for entities to see if their Fedora counterpart exists, and if so, add a Link header to the Fedora representation.  This probably shouldn't be exposed to `anonymous` users.

**Comment by ajs6f** [comment]
This should definitely be configurable, because it represents a _massive_ security hole.

**Comment by dannylamb** [comment]
Yeah, it'd definitely need to get locked down.  The usual Drupal technique for something that has security implications is to give it its own special permission and put a big warning next to the checkboxes like this: 
![screenshot from 2018-07-24 13-05-22](https://user-images.githubusercontent.com/20773151/43151419-491cec7e-8f42-11e8-9248-b13904518a57.png)

**Comment by ajs6f** [comment]
Perfect!

**Comment by dannylamb** [comment]
As of https://github.com/Islandora-CLAW/Crayfish-Commons/releases/tag/0.0.2, you can `composer require islandora/crayfish-commons` to get a Gemini client.
